libs/agno/agno/team/team.py

```diff
@@ -3137,10 +3138,10 @@ def _get_member_name(self, entity_id: str) -> str:
         for member in self.members:
             if isinstance(member, Agent):
                 if member.agent_id == entity_id:
-                    return member.name or entity_id
+                    return url_safe(member.name) or entity_id
```

Comment:
We don't want to make this url_safe? This is for the print_response logic where we want the raw name